meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -353,6 +353,20 @@ LMP_FLASHLAYOUT_stm32mp1-eval = "FlashLayout_emmc_stm32mp157a-ev1-trusted.tsv"
 # Xilinx
 USE_XSCT_TARBALL_zynqmp = "1"
 
+# TI AM64x
+PREFERRED_PROVIDER_virtual/kernel_am64xx-sk = "linux-ti-staging"
```

Comment:
Can you check if the machine is already setting this? See with bitbake -e and look for PREFERRED_PROVIDER_virtual/kernel_am64xx-sk.

Reply:
for grep "^PREFERRED_PROVIDER_virtual/kernel" I get linux-lmp and nothing set for am64xx-sk without it bing in lmp-machine-custom.inc

Reply:
To clarify:
```
# $PREFERRED_PROVIDER_virtual/kernel [12 operations]
#   set? /home/scottml/fio/lmp-master/build-lmp-base/conf/local.conf:56
#     "linux-lmp"
#   set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-ti/conf/machine/include/k3.inc:13
#     "linux-ti-staging"
#   set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-base/conf/distro/include/lmp.inc:16
#     "linux-lmp"
#   override[rpi]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:18
#     "linux-lmp-rpi"
#   override[corstone700]:set /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:74
#     "linux-lmp"
#   override[mx6ull]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:201
#     "linux-lmp-fslc-imx"
#   override[apalis-imx6]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:238
#     "linux-lmp"
#   override[apalis-imx8]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:264
#     "linux-lmp-toradex-imx"
#   override[mx8mq]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:290
#     "linux-lmp-fslc-imx"
#   override[mx8mm]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:321
#     "linux-lmp-fslc-imx"
#   override[stm32mp1common]:set? /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:333
#     "linux-lmp-stm32"
#   override[am64xx-sk]:set /home/scottml/fio/lmp-master/build-lmp-base/conf/../../layers/meta-lmp/meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:357
#     "linux-ti-staging"
# pre-expansion value:
#   "linux-ti-staging"
PREFERRED_PROVIDER_virtual/kernel="linux-ti-staging"
```